python-frontend/typeshed_serializer/serializer/symbols.py

```diff
@@ -337,7 +338,8 @@ def __init__(self, name: str, fullname: str, is_imported_module=False, type_desc
 
     @classmethod
     def from_var(cls, var: mpn.Var, name: str = None):
-        return cls(var.name if name is None else name, var.fullname, type_descriptor=TypeDescriptor(var.type) if var.type else None)
+        return cls(var.name if name is None else name, var.fullname,
```

Comment:
is this change intended?

Reply:
It's just a formatting change because the line was too long I assume. I don't remember I did it to be honest though, and it could have been separated from actual changes, but now that it's here I guess it can stay?